### ISSUE TITLE
Make a release

### ISSUE THREAD

**Issue by bertfrees** [daisy/pipeline-tasks#100]
- [x] Finalize the release notes
- [x] Tag the assembly
- [x] Merge all the release branches, push tags and close issues
  - [x] [jstyleparser](https://github.com/daisy/braille-css/releases/tag/1.13.0)
  - [x] [xspec-maven-plugin](https://github.com/daisy/xspec-maven-plugin/releases/tag/xspec-runner-1.0.2)
  - [x] [xproc-maven-plugin](https://github.com/daisy/xproc-maven-plugin/releases)
  - [x] [pipeline-build-utils](https://github.com/daisy/pipeline-build-utils/releases)
  - [x] [pipeline-framework](https://github.com/daisy/pipeline-framework/releases/tag/v1.10.3)
  - [x] [pipeline-modules-common](https://github.com/daisy/pipeline-modules-common/releases/tag/v1.10.3)
  - [x] [pipeline-scripts-utils](https://github.com/daisy/pipeline-scripts-utils/releases/tag/v1.10.2)
  - [x] [pipeline-mod-braille](https://github.com/daisy/pipeline-mod-braille/releases/tag/v1.10.1)
  - [x] [xmlcalabash1](https://github.com/daisy/xmlcalabash1/releases/tag/1.1.9-p1-95)
  - [x] [pipeline-mod-tts](https://github.com/daisy/pipeline-mod-tts/releases/tag/v1.10.2)
  - [x] [pipeline-updater-gui](https://github.com/daisy/pipeline-updater-gui/releases/tag/v1.0.1)
  - [x] [pipeline-assembly](https://github.com/daisy/pipeline-assembly/releases)
- [x] Pull everything into super project
- [x] Close all the v1.10.2 milestones
  - [x] [pipeline](https://github.com/daisy/pipeline/milestones)
  - [x] [pipeline-assembly](https://github.com/daisy/pipeline-assembly/milestones)
  - [x] [pipeline-build-utils](https://github.com/daisy/pipeline-build-utils/milestones)
  - [x] [pipeline-framework](https://github.com/daisy/pipeline-framework/milestones)
  - [x] [pipeline-gui](https://github.com/daisy/pipeline-gui/milestones)
  - [x] [pipeline-it](https://github.com/daisy/pipeline-it/milestones)
  - [x] [pipeline-mod-audio](https://github.com/daisy/pipeline-mod-audio/milestones)
  - [x] [pipeline-mod-braille](https://github.com/daisy/pipeline-mod-braille/milestones)
  - [x] [pipeline-mod-nlp](https://github.com/daisy/pipeline-mod-nlp/milestones)
  - [x] [pipeline-mod-tts](https://github.com/daisy/pipeline-mod-tts/milestones)
  - [x] [pipeline-modules-common](https://github.com/daisy/pipeline-modules-common/milestones)
  - [x] [pipeline-scripts](https://github.com/daisy/pipeline-scripts/milestones)
  - [x] [pipeline-scripts-utils](https://github.com/daisy/pipeline-scripts-utils/milestones)
  - [x] [pipeline-tasks](https://github.com/daisy/pipeline-tasks/milestones)
  - [x] [pipeline-updater-gui](https://github.com/daisy/pipeline-updater-gui/milestones)
- [x] Build the distribution packages
- [x] Update the updater's release descriptor
- [x] Upload the distribution to GitHhub and publish the release
- [x] Send the announcement to the lists
- [x] Update website

**Comment by bertfrees** [comment]
See https://github.com/daisy/pipeline-assembly/pull/133

**Comment by rdeltour** [comment]
Closing as invalid, since we're doing 1.10.2.

**Comment by bertfrees** [comment]
Why "invalid"?

**Comment by rdeltour** [comment]
> Why "invalid"?

We need a label to say that the issue is cancelled (this issue was for an intermediary release, before what is now 1.10.2.
I'm using `invalid` or `wontfix` to state that, but I'm happy to use another label if you have a better idea.

**Comment by rdeltour** [comment]
I edited some of my notes to start a [release guide](https://gist.github.com/rdeltour/4c4ef6c1b50a277f1135b75d23d698c2).

**Comment by bertfrees** [comment]
Great.

**Comment by bertfrees** [comment]
I can help you a bit finishing up. I'll start with
- Merge all the release branches, push tags and close issues
- Pull everything into super-project

This part isn't really critical (except maybe it's needed to close the v1.10.2 milestones?) but has to happen anyway. Building and uploading the packages, publishing the release on Github and sending the announcements is more urgent and can be done in parallel.
